### PR TITLE
Update parser comment with es7 grammar

### PR DESCRIPTION
**Pull request by yuit**
https://tc39.github.io/ecma262/#prod-ExponentiationExpression

### CODE REVIEW

File: Gulpfile.ts [hunk 1]
```diff
@@ -551,7 +551,7 @@ function restoreSavedNodeEnv() {
     process.env.NODE_ENV = savedNodeEnv;
 }
 
-let testTimeout = 20000;
+let testTimeout = 40000;
```
Comment (DanielRosenwasser) [inline Gulpfile.ts]: Is this related?

File: src/compiler/parser.ts [hunk 2]
```diff
@@ -3373,24 +3373,44 @@ namespace ts {
         }
 
         /**
-         * Parse ES7 unary expression and await expression
+         * Parse ES7 exponential expression and await expression
+         *
+         * ES7 ExponentiationExpression:
+         *      1) UnaryExpression[?Yield]
+         *      2) UpdateExpression[?Yield]**ExponentiationExpression[?Yield]
```
Comment (DanielRosenwasser) [inline src/compiler/parser.ts]: Nit: spaces around `**`

File: Gulpfile.ts (hunk 1 above)
Comment (yuit) [inline Gulpfile.ts]: opps I am using mac and the machine time out. I can roll back though would love to keep it so I can run on my mac :smile_cat:
